hack/e2e/releases.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+function get_latest_release() {
+  set +x
+  if [ -z "${GITHUB_TOKEN:-}" ]; then
```

Comment:
I'm not sure if the `GITHUB_TOKEN` will actually be set when we do this in CAPM3. Worth checking!

Reply:
I think this var always exported from project infra https://github.com/metal3-io/project-infra/blob/9470543342be88a38a94e120b81ee04c90a9576b/jenkins/scripts/files/run_integration_tests.sh#L6